### ISSUE TITLE
Pairs not returning value causes error

### ISSUE THREAD

**Issue by neochine** [nixonjoshua98/dexscreener#5]
https://photon-sol.tinyastro.io/en/lp/4hHPXcz54CyopQbqxyZtUYkAmrgwHCfjnVrxRewyELHN?handle=25412d37a880ec0876ec
For this token BONKKILLER, pair address is 4hHPXcz54CyopQbqxyZtUYkAmrgwHCfjnVrxRewyELHN but when its used to fetch value, it results in error
```
>>> screener.get_token_pair('solana', '4hHPXcz54CyopQbqxyZtUYkAmrgwHCfjnVrxRewyELHN')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.9/dist-packages/dexscreener/client.py", line 22, in get_token_pair
    return TokenPair(**resp["pair"])
TypeError: dexscreener.models.TokenPair() argument after ** must be a mapping, not NoneType
```

**Comment by neochine** [comment]
PR: https://github.com/nixonjoshua98/dexscreener/pull/6

**Comment by nixonjoshua98** [comment]
Fixed by #6  in V1.0
